__tests__/materials/material.mutation.test.js

```diff
@@ -1,427 +1,415 @@
+    expect(newMaterial).toHaveProperty('statusCode', 400);
   });
-  describe('update material tests', () => {
-    test('#3 update material', async () => {
-      const res = await client
-        .mutate({
-          mutation: gql`
-            mutation($id: ID!, $material: MaterialInput!) {
-              updateMaterial(id: $id, material: $material) {
-                ... on Material {
-                  name {
-                    lang
-                    value
-                  }
-                  description {
-                    lang
-                    value
-                  }
-                  purpose
 
-                  colors {
-                    code
-                    name {
-                      lang
-                      value
-                    }
-                    simpleName {
-                      lang
-                      value
-                    }
-                    available
-                    images {
-                      large
-                      medium
-                      small
-                      thumbnail
-                    }
-                  }
-                  additionalPrice {
-                    currency
-                    value
-                  }
-                  available
+  it('#3 should update material', async () => {
+    const res = await operations.mutate({
+      mutation: gql`
+        mutation($id: ID!, $material: MaterialInput!) {
+          updateMaterial(id: $id, material: $material) {
+            ... on Material {
+              name {
+                lang
+                value
+              }
+              description {
+                lang
+                value
+              }
+              purpose
+
+              colors {
+                code
+                name {
+                  lang
+                  value
+                }
+                simpleName {
+                  lang
+                  value
                 }
-                ... on Error {
-                  message
+                available
+                images {
+                  large
+                  medium
+                  small
+                  thumbnail
                 }
               }
+              additionalPrice {
+                currency
+                value
+              }
+              available
+            }
+            ... on Error {
+              message
             }
-          `,
-          context: { headers: { token } },
-          variables: { id: materialId, material: materialToUpdate },
-        })
-        .catch(e => e);
-      const updatedMaterial = res.data.updateMaterial;
+          }
+        }
+      `,
+      variables: {
+        id: materialId,
+        material: materialToUpdate,
+      },
+    });
+
+    const updatedMaterial = res.data.updateMaterial;
+
+    expect(updatedMaterial).toBeDefined();
+    expect(updatedMaterial).toHaveProperty(
+      'name',
+      materialToUpdate.name.map(item => ({
```

Comment:
Please, simplify it to the `materialToUpdate.name` because mapping is useless now due to the returning the same object